IDE/Espressif/ESP-IDF/examples/wolfssl_client/main/time_helper.c

```diff
@@ -0,0 +1,333 @@
+/* time_helper.c
```

Comment:
This (time_helper.c) and wifi_connect.c look to be duplicated with wolfssl_client and wolfssl_server. Not critical, and could be approved with out changing, but please take a look at combining them for ease of maintenance. Having the client and server use the same time_helper.c and wifi_connect.c files.

Reply:
Hi @JacobBarthelmeh, you are correct: the code is duplicated between both the client and server applications. It was always duplicated in the `main.c`, but now it is more obvious in separate files.

I prefer to have the examples ready-to-run, fresh out of the source code, with no additional steps for install/setup (well, other than setting up the external ESP-IDF environment). These are also examples that are published as-is to the Espressif component registry. So it seems best to have the duplicate source code in order to have each example be stand-alone.

I do agree with the duplication and maintenance annoyance. @bandi13 also pointed that out in a recent PR of mine. 

Other than moving it to something like `esp_util.c` in wolfCrypt (where it arguably may not belong), do you have a suggestion where "_common-but-not-wolfssl_" code like this should live? Perhaps just move both the client and server to a shared directory with two separate projects but otherwise common code? This would be different than all the other examples. Completely open to suggestions here.